backend/src/api/link/interface/link.interface.ts

```diff
@@ -1,8 +1,10 @@
-import {IIdHaver} from "../../../APIHelpers/IIdHaver";
+import { ParticipantInterface } from 'src/api/participant/interfaces/participant.interface';
+import { participantEntity } from 'src/entity/participant.entity';
+import { IIdHaver } from '../../../APIHelpers/IIdHaver';
 
 export interface LinkInterface extends IIdHaver {
-    participant_id?: string;
-    used?: boolean;
-    sent_at?: string;
-    link?: string;
-}
\ No newline at end of file
+  participant?: ParticipantInterface;
```

Comment:
Here you are using ParticipantInterface not participan entity. Choose one and stick to it